hosters/hetzner-dedicated/hetzner-dedicated-wipe-and-install-nixos.sh

```diff
@@ -30,13 +24,90 @@
 #   being able to login without any authentication.
 # * The script reboots at the end.
 
-set -eu
-set -o pipefail
 
-set -x
+# Default options
+RAIDLEVEL=1
+HOSTNAME='hetzner'
+NETWORK_BRIDGE=0
+
+# Default SSH key
+SSH_PUBKEY="$(xargs < <(find "$HOME/.ssh" -name '*.pub' -exec cat {} \; -quit))"
+
+# Strict mode
+set -euo pipefail
+
+# Parameter parsing
+while true; do
+  case "$1" in
```

Comment:
Not sure why but I get "unbound variable" here on Hetzner AZ101 Linux rescue system. Had to delete the entire parameter parsing code to proceed.